### PR TITLE
Update code for deprecations and replace batchnorm

### PR DESCRIPTION
**Pull request by jrock08**
Just to highlight an issue, don't actually want to merge this.

### CODE REVIEW

**Comment by jrock08** [comment]
Ok, this is now a reasonable merge request, not just highlighting an issue.

File: tests/test_tftools.py [hunk 1]
```diff
@@ -172,9 +201,9 @@ def test_batchnorm_train_mode():
     assert (np.all(y_eval1 == np.zeros(input_shape))), assert_str
     assert_str = 'batch mean and var are not used correctly' + \
                  'during training with batch norm'
-    assert (np.all(y_eval2 == np.zeros(input_shape))), assert_str
-    assert_str = 'ema mean is not updated during training with batch norm'
-    assert (not np.all(ema_mean_eval1 == ema_mean_eval2)), assert_str
+    #assert (np.all(y_eval2 ==np.zeros(input_shape))), assert_str
```
Comment (BigRedT) [inline tests/test_tftools.py]: delete the comments

File: requirements.txt [hunk 2]
```diff
@@ -4,4 +4,3 @@ pillow
 pytest
 tensorflow
 future
-scipy
```
Comment (BigRedT) [inline requirements.txt]: actually scipy is used in placeholder management for sparse matrices

Comment (jrock08) [inline requirements.txt]: ok, added back, any idea why it doesn't complain when we test it here?

**Pull request merged**